engine/src/main/scala/cromwell/server/CromwellRootActor.scala

```diff
@@ -109,6 +110,9 @@ abstract class CromwellRootActor(terminator: CromwellTerminator,
   lazy val ioActor = context.actorOf(IoActor.props(LoadConfig.IoQueueSize, nioParallelism, gcsParallelism, Option(ioThrottle), serviceRegistryActor), "IoActor")
   lazy val ioActorProxy = context.actorOf(IoActorProxy.props(ioActor), "IoProxy")
 
+  // Register the IoActor with the service registry:
+  serviceRegistryActor ! IoActorRef(ioActorProxy)
```

Comment:
why this and the one above? Will this not lead to Programmer Error admonishments?

Reply:
🤦‍♂

Reply:
Sometimes I like to make sure my reviewers are on their toes...

Reply:
Fixed by removing the `IoActor`'s self registration step